logs-shipper/src/test/java/com/ResourceTest.java

```diff
@@ -0,0 +1,52 @@
+package com;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.net.HttpURLConnection;
+
+public class ResourceTest {
+
+    public static String REST_API_URI = "http://localhost:8080/api/";
+
+    public static WebTarget getWebTarget() {
+        return ClientBuilder.newClient().target(REST_API_URI);
```

Comment:
You can create it once inside a `@BeforeClass` method and assign to a private member